source/config/201-clusterrole.yaml

```diff
@@ -97,7 +97,7 @@ rules:
 kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: eventing-redis-source-observer
+  name: eventing-sources-source-observer
```

Comment:
this should be `eventing-redis-source-observer`

Reply:
Today, we have a [source conformance test](eventing-sources-source-observer) that checks for this particular name and if not, fails the test. For a source, it expects only a clusterrole with name `eventing-sources-source-observer` to be valid and its rules be added to `source-observer` role.